packages/xo-web/src/common/render-xo-item.js

```diff
@@ -236,7 +237,7 @@ const xoItemToRender = {
   network: network => <PoolObjectItem object={network} />,
 
   // SR.
-  SR: ({ id }) => <SrItem id={id} />,
+  SR: ({ id }, resourceSet) => <SrItem id={id} resourceSet={resourceSet} />,
```

Comment:
You can create a new type `'SR-resourceSet': ({ id }) => <SrResourceSetItem id={id} />`.